dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/CrudControllerAdvice.java

```diff
@@ -248,6 +267,10 @@ private String getGenericFieldErrorMessage( String fieldType )
 
     private String getNotValidValueMessage( Object value, String field )
     {
+        if ( value == null || "".equals( value ) )
```

Comment:
You might want to follow GitHubs advice. See code scanning notice above 😄

Reply:
I was reluctant because I needed to cast to String but I followed github advice